inference-engine/src/gna_plugin/frontend/layer_quantizer.hpp

```diff
@@ -642,7 +642,11 @@ class DataQuantizer<Desc, InferenceEngine::WeightableLayer *> : public DataQuant
  public:
     explicit DataQuantizer(float scaleFactor) : DataQuantizerBase(scaleFactor) {}
     bool operator()(InferenceEngine::WeightableLayer *wl) const {
-        (*this)(wl, typename Desc::MandatoryType());
+        if (LayerInfo(wl).isConcatAlignFilter()) {
```

Comment:
Also ConcatAlignFilter isn't the only weightable layer which can be not quantized by POT. GNA plugin also inserts ScaleShift, ConvolutionFilter. Or FakeQuantize layer are missed for some layers in the original network by some reason (probably we should print warnings for such cases but handle them with some optional type).
Shouldn't we use Optional type for all layers which don't have FakeQuantizes for their weights?